packages/db/src/core/integration/index.ts

```diff
@@ -111,6 +114,10 @@ function astroDBIntegration(): AstroIntegration {
 				});
 			},
 			'astro:build:start': async ({ logger }) => {
+				if(!connectToStudio && !databaseFileEnvDefined() && output === 'server') {
```

Comment:
```suggestion
				if(!connectToStudio && !databaseFileEnvDefined() && (output === 'server' || output === 'hybrid')) {
```
Don't forget hybrid output!

Reply:
I think we should error on hybrid, because the server routes might be using db, so we should err on the side of caution so they don't accidentally deploy something where only some routes fail.

Reply:
Wait, I thought the code _did not_ error on hybrid unless we make this change. Are you saying we **should** error if you use hybrid mode? Or saying it **shouldn't** error?

Reply:
Oh you're right, i'm saying it should, but it wasn't currently. I'll update.

Reply:
Updated